tests/prodigy/test_prodigy_entrypoints.py

```diff
@@ -0,0 +1,20 @@
+from deep_reference_parser.prodigy import annotate_numbered_references
+from deep_reference_parser.prodigy import prodigy_to_tsv
+from deep_reference_parser.prodigy import reach_to_prodigy
+from deep_reference_parser.prodigy import reference_to_token_annotations
+
+
+def test_annotate_numbered_references_entrypoint():
+    pass
```

Comment:
do these need to be filled before merging?

Reply:
> Code looks good. Just waiting to run the examples. Takes some time to download the weights for some reason.

Yeah they are 300mb  :speak_no_evil:

Reply:
> do these need to be filled before merging?

Yes good spot!

Reply:
Resolved in https://github.com/wellcometrust/deep_reference_parser/pull/10/commits/2e0bf61c5dd18fda6a5714c6a0cdf9956efafb58